mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -20,7 +20,7 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	FIELD field_7508 lastPlayedLevelUpSoundTime I
 	FIELD field_7510 experienceProgress F
 	FIELD field_7511 SCORE Lnet/minecraft/class_2940;
-	FIELD field_7512 container Lnet/minecraft/class_1703;
+	FIELD field_7512 screenHandler Lnet/minecraft/class_1703;
```

Comment:
My problem with `screenHandler`, `menu` or `screenController` in `PlayerEntity` is I expect it to be used for also other things that handles screen/menu without items

Reply:
Don't think there's a way around this unless we go with `handledScreenHandler` which sounds pretty dumb.

Reply:
`currentScreenHandler`, maybe? Would be like `currentScreen` in MinecraftClient